testdata/go/v3/memcached-operator/Makefile

```diff
@@ -130,13 +137,58 @@ rm -rf $$TMP_DIR ;\
 }
 endef
 
-.PHONY: bundle ## Generate bundle manifests and metadata, then validate generated files.
-bundle: manifests kustomize
+.PHONY: bundle
+bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests --interactive=false -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	operator-sdk bundle validate ./bundle
 
-.PHONY: bundle-build ## Build the bundle image.
-bundle-build:
+.PHONY: bundle-build
+bundle-build: ## Build the bundle image.
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+
+.PHONY: bundle-push
+bundle-push: ## Push the bundle image.
+	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
+
+.PHONY: opm
+OPM = ./bin/opm
+opm: ## Download opm locally if necessary.
+ifeq (,$(wildcard $(OPM)))
+ifeq (,$(shell which opm 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPM)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$${OS}-$${ARCH}-opm ;\
+	chmod +x $(OPM) ;\
+	}
+else
+OPM = $(shell which opm)
+endif
+endif
```

Comment:
$ make opm
make: Nothing to be done for 'opm'.

$ opm
-bash: opm: command not found

should not work

Reply:
Run
```sh
rm -rf ./bin
make opm
./bin/opm version
```
what do you see?

Reply:
it is there: 

```$ ls
controller-gen	kustomize	opm
```

Then, by removing and make opm again: 

```
$ ./bin/opm version
Version: version.Version{OpmVersion:"v1.15.1", GitCommit:"f83ace4", BuildDate:"2020-11-04T15:46:08Z", GoOs:"darwin", GoArch:"amd64"}
```

However:

```
$ make catalog-build
./bin/opm index add --mode semver --tag my.domain/testcas-catalog:v0.0.1 --bundles my.domain/testcas-bundle:v0.0.1 
INFO[0000] building the index                            bundles="[my.domain/testcas-bundle:v0.0.1]"
Error: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
Usage:
  opm index add [flags]
```

Reply:
i think we found my issue in the other comment 👍